CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## v1.7.2
+
+* Bux fixes
+  * Add TAR option `--no-same-owner` to fix issues with tar files containing
+    large UIDs.
```

Comment:
"to fix errors when untarring artifacts as the root user"

Reply:
Updated